### PR TITLE
Fixes truncation of plain circuit drawers when used with matplotlib

### LINKED ISSUE #5393: Chipped figure in matplotlib for a plain quantum circuit
### Information

- **Qiskit Terra version**: 0.16.1
- **Python version**: 3.8.6
- **Operating system**: Artix Linux

### What is the current behavior?
When drawing a plain quantum circuit (just initializing the quantum and classical registers without implementing any gates), the figure rendered in matplotlib renders chipped text. Testing this with a previous release of terra (I used 0.15.1), the text appears fine without any chipping whatsoever.

### Steps to reproduce the problem
```python
from qiskit import *

qr = QuantumRegister(2)
cr = ClassicalRegister(3)
qc = QuantumCircuit(qr,cr)

qc.draw('mpl')
```

*With current release (0.16.1)*:
Figure_3

*With a previous release (0.15.1)*:
Figure_1

### What is the expected behavior?
The expected behaviour is to draw the circuit without any text chipping and draw the circuit as implemented in version 0.15.1.

### PR DESCRIPTION
**Pull request by paniash**
### Summary
This PR fixes #5393.

## TODO:
- ~Fix the issue with extended lines when drawn with gates.~ Fixed.

### CODE REVIEW

File: qiskit/visualization/matplotlib.py [hunk 1]
```diff
@@ -1059,7 +1059,10 @@ def _draw_ops(self, verbose=False):
             self._xmax = self._fold + 1 + self._x_offset - 0.9
             self._ymax = (n_fold + 1) * (self._n_lines + 1) - 1
         else:
-            self._xmax = max_anc + 1 + self._x_offset - 0.9
+            if self._ops == []:
+                self._xmax = max_anc + 1 + self._x_offset - 0.4
+            else:
+                self._xmax = max_anc + 1 + self._x_offset - 0.9
```
Comment (enavarro51) [inline qiskit/visualization/matplotlib.py:1065]:
```
            x_incr = 0.4 if not self._ops else 0.9
            self._xmax = max_anc + 1 + self._x_offset + x_incr
```
Looks good. Typically we use `if not self._ops` in this situation and what I have here is a bit more concise.

Comment (paniash) [reply qiskit/visualization/matplotlib.py]: Thanks! I've made the changes. :)

**Comment by 1ucian0** [comment]
Dont forget to check the tests here https://mybinder.org/v2/gh/paniash/qiskit-terra/chipped?urlpath=apps/test/ipynb/mpl_tester.ipynb

Also, you might want to add a test based on https://github.com/Qiskit/qiskit-terra/issues/5393

**Comment by paniash** [comment]
> Dont forget to check the tests here https://mybinder.org/v2/gh/paniash/qiskit-terra/chipped?urlpath=apps/test/ipynb/mpl_tester.ipynb

All green!

> Also, you might want to add a test based on #5393

Pardon but I'm not sure if I understand this. Could you elaborate?

**Comment by 1ucian0** [comment]
> > Also, you might want to add a test based on #5393
>
> Pardon but I'm not sure if I understand this. Could you elaborate?

No worries, just added in https://github.com/Qiskit/qiskit-terra/pull/5421/commits/2dd7cbbb24a426148a076000baa3e7cc2fbcaffe in case you are curious

**Comment by paniash** [comment]
> No worries, just added in [2dd7cbb](https://github.com/Qiskit/qiskit-terra/commit/2dd7cbbb24a426148a076000baa3e7cc2fbcaffe) in case you are curious

Thanks! I'll keep this in mind when I open another PR. :)

**Review by 1ucian0 — APPROVED** [review]

**Comment by 1ucian0** [comment]
Thanks and welcome to the community!

**Comment by paniash** [comment]
@1ucian0 @enavarro51 Thank you very much for all your help! Glad to be a part of the community. :)

**Pull request merged**